universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,32 @@
+# --- JSON-based tables
+
+# --- !Ups
+create table applicants (
+  id bigserial primary key,
+  applicant json not null
```

Comment:
Let's rename this or the table. The ORM expects a class named `Applicant` to represent entries in this table. It'd be nice to have another class for containing the top-level applicant data and for it to have a name that corresponds to the column's name.

Maybe call the column `answer`(s)?

Reply:
I have named all the columns `object`, which feels like an okay solution to me - `Applicant.object` is how I'd want to refer to that, I think.  LMK what you think.

Reply:
LGTM